contracts/modules/royalty/RoyaltyModule.sol

```diff
@@ -175,6 +182,10 @@ contract RoyaltyModule is
         RoyaltyModuleStorage storage $ = _getRoyaltyModuleStorage();
         if (!$.isWhitelistedRoyaltyToken[token]) revert Errors.RoyaltyModule__NotWhitelistedRoyaltyToken();
 
+        IDisputeModule dispute = IDisputeModule($.disputeModule);
+        if (dispute.isIpTagged(receiverIpId)) revert Errors.RoyaltyModule__IpIsTagged();
+        if (dispute.isIpTagged(payerIpId)) revert Errors.RoyaltyModule__IpIsTagged();
```

Comment:
Can we combine these into one if, since it's reverting with the same error message

Reply:
https://github.com/storyprotocol/protocol-core-v1/pull/35/commits/213e2b7ad4314abc59b273e79d364c384a80499f